slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/model/TestFileModel.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import dsl.InterchangeV5Format;
+
+public class TestFileModel {
+
+  final String name;
+  final boolean shouldSucceed;
+  final String genesis_validators_root;
+
+  final InterchangeV5Format interchangeContent;
+
+  final List<BlockTestModel> blocks;
+  final List<AttestionTestModel> attestations;
+
+  public TestFileModel(
```

Comment:
do they require JsonCreator annotation as well? (assuming you are deseriaizing them)

Reply:
yeah I am - but didn't seem to complain - have added it in.